torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -481,6 +481,8 @@ std::string encodeRHS(Node * n) {
     {aten::add, "${0} + ${2}*${1}"},
     {aten::sub, "(${0} - ${2}*${1})"},
     {aten::rand_like, "uniform(rnd())"},
+    //min, max
+    {aten::clamp, "fmaxf(fminf(${0}, ${2}), ${1})"},
```

Comment:
I did not look up how clamp is implemented in the core, but this won't propagate nans (fmaxf and fminf return other when comparing nan with other according to standard) and we usually try to propagate nans. A few lines above, aten::relu is implemented as `{aten::relu, "${0} < 0 ? 0.f : ${0} "},` for this reason.

Reply:
Well we don't need to maintain the same exact NaN semantics in the JIT I think (it's kind of UB for most of our ops), but we don't need to use the intrinsics, because they might end up compiling to code slower than one that uses simple comparisons.

Reply:
While there might be code not providing proper NaN semantics, I'm kind of attached to them.
For clamp, we actually have min/max=NaN to mean "unbounded", so `(a > max ? max : a)` gives the right thing.